Vexole/findadoctor-fe
# Login Flow

This PR contains:

- Login
- Register
- Forgot Password
- Reset Password
- Logout
- Email Confirmations
- Fix some errors
- Improve structure
- Improve interfaces

How to test:

npm i 
npm run dev

Disclaimer: To test confirm email and reset password features, you have to set the url manually.
E.g.: 
http://localhost:3000/auth/confirm-email?userId={UserIdFromEmail}&token={TokenFromEmail}
http://localhost:3000/auth/reset-password?userId={UserIdFromEmail}&token={TokenFromEmail}
